docs/pages/api-reference/client/commit.md

```diff
@@ -34,6 +34,11 @@ it only works with real client/server and mock client/server ignore it.
 :::
 </Expandable>
 
+<Expandable title="incremental" type="bool" defaultVal="False">
+If set to True, server tries to add only requested datasets and featuresets assuming 
+everything else remains same.
+</Expandable>
```

Comment:
add the additiona datasets and featuresets to the existing datasets and features already committed. 

Also mention that you are not allowed to make any changes to existing datasets / featuresets in this mode

Reply:
added